arbitrum-docs/stylus/concepts/stylus-cache-manager.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Stylus caching strategy'
+description: 'A conceptual overview of the Stylus caching strategy and CacheManager contract, and explaining its functionality.'
+sme: mahsa-moosavi
+target_audience: 'Developers deploying smart contracts using Stylus.'
+sidebar_position: 3
+---
+
+Stylus is designed for fast computation, but it faces significant performance issues due to the slow activation process when entering a contract. This slow [activation](https://docs.arbitrum.io/stylus/stylus-gentle-introduction#activation) takes up substantial resources and time. These repeated activations lead to inefficiencies and hinder the overall performance of the system. To address this problem, we have implemented a caching strategy. The cache serves as a bypass, storing frequently accessed contracts in-memory. This approach allows for avoiding repeated activations, thereby saving resources and time while significantly improving the speed and efficiency of contract execution.
```

Comment:
Activation is different. You only need to activate a program once and then it stays "active" for a year or until stylus version changes. It's unrelated to caching.
Here we're talking about something else - in some places it's called "initialization" in this document, which is a good name for it.
Most places where you describe "activation" in this doc should be changed to "initialization"